### ISSUE TITLE
operation error S3: PutObject - api error AccessDenied: Access Denied

### ISSUE THREAD

**Issue by thebillington** [sst/ion#1005]
I'm running into an error with initial deployment of a new Next app: [github repo](https://github.com/thebillington/ecoworks-monitoring/tree/sst)

`sst version == sst 3.0.102`

I am logged into `aws-cli` with an `IAM Identity Center` account via SSO. The account has the `AdminAccess` IAM permission.

When deploying, I'm getting the following error:

```
OpenNext build complete.
|  Updated     EcoworksMonitoring sst:aws:Nextjs → EcoworksMonitoringBuild sst:Run (24.1s)
|  Error       EcoworksMonitoring sst:aws:Nextjs → EcoworksMonitoringAssetFiles sst:aws:BucketFiles
|  operation error S3: PutObject, https response error StatusCode: 403, RequestID: ..., HostID: ..., api error AccessDenied: Access Denied
|  Created     EcoworksMonitoring sst:aws:Nextjs → EcoworksMonitoringRevalidationSeederCode aws:s3:BucketObjectv2
|  Created     EcoworksMonitoring sst:aws:Nextjs → EcoworksMonitoringRevalidationEventsSubscriberBasbbmFunctionCode aws:s3:BucketObjectv2a
```

`sst.config.ts`

```
export default $config({
  app(input) {
    return {
      name: "ecoworks-monitoring",
      home: "aws",
      removal: "remove",
      providers: {
        aws: {
          region: "eu-west-2",
          profile: "aws-personal"
        }
      }
    };
  },
  async run() {
    new sst.aws.Nextjs("EcoworksMonitoring");
  },
});
```

`~/.aws/config`

```
[profile aws-personal]
sso_session = aws-personal
sso_account_id = ...
sso_role_name = AdministratorAccess
region = eu-west-2
output = json
[sso-session aws-personal]
sso_start_url = ...
sso_region = eu-west-2
sso_registration_scopes = sso:account:access
```

I'm struggling to debug this. I've tried the `--verbose` flag for the deployment but it doesn't provide any more insight.

Any help is much appreciated. Thanks!

**Comment by thebillington** [comment]
I appreciate that these issues take time to prioritise and review.

This is preventing the deployment of my current project via `sst`. Do you have any suggestions for things I could try to work around or fix the issue?

Today I have updated to the `--latest` version of `sst = 3.1.10` and am still experiencing the same problem.

**Comment by thebillington** [comment]
Managed to fix this today after some more debugging.

When I initially deployed I had used a different app name and it looks like there was a conflict.

I manually deleted the `asset` and `state` buckets via the AWS console and then had to delete the `/sst/bootstrap` parameter from parameter store in order to force `sst` to deploy everything from scratch.

I also changed the project name to `ecoworks-monitoring-system`.

Let me know if there's any more info I can add to help with preventing the issue going forward.
